features/fields-filtered-PersoonBeperkt.csv

```diff
@@ -0,0 +1,20 @@
+pad
+adressering
+adressering.adresregel1
+adressering.adresregel2
+adressering.adresregel3
+adressering.land
+burgerservicenummer
+geboorte
+geboorte.datum
+geslacht
+leeftijd
+naam
+naam.adellijkeTitelPredicaat
+naam.adellijkeTitelPredicaat.soort
```

Comment:
dit veld moet volgens mij weg omdat dit eensub-veld is van een waardetabel

Reply:
gecorrigeerd